R/data-repository.R

```diff
@@ -0,0 +1,56 @@
+#' @title DataRepository
+#' @docType class
+#' @description  An object typically holding observed data
+#' @format NULL
+DataRepository <- R6::R6Class(
+  "DataRepository",
+  inherit = DotNetWrapper,
+  cloneable = FALSE,
+  active = list(
+    #' @field baseGrid Returns the base column for the population (typically time column)

```

Comment:
Why `population`?

Reply:
Yes sorry. All descriptions were copied paste so that I could get going. Still WIP